<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../img/img.qrc">:/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Zucker">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Zucker</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Milch">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Milch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../img/img.qrc"/>
 </resources>
 <connections/>
</ui>
